modules/gnome/hm.nix

```
@@ -1,118 +1,129 @@
 {
+  lib,
   pkgs,
   config,
-  lib,
+  mkTarget,
   ...
 }:
+mkTarget {
+  name = "gnome";
+  humanName = "GNOME";
+  autoEnable = pkgs.stdenv.hostPlatform.isLinux;
```

Following commit 289356d03169 ("treewide: use `mkEnableTargetWith` for dynamic conditions") and the

https://github.com/nix-community/stylix/blob/e7fa0e5cc2336b6b25310d5e49c149f14fdbc1bb/stylix/mk-target.nix#L71-L75

documentation, the `autoEnableExpr` attribute must be specified:

```suggestion
  autoEnable = pkgs.stdenv.hostPlatform.isLinux;
  autoEnableExpr = "pkgs.stdenv.hostPlatform.isLinux";
```